### ISSUE TITLE
Livechat Browser Cache Problem

### ISSUE THREAD

**Issue by robinreinhardt** [RocketChat/Rocket.Chat#13507]
### Description:
The information (name and email address) which can be required before you can start a conversation is cached by the browser.

### Steps to reproduce:

1. Click on the livechat element
2. Enter your name and email address
3. Start a conversation
4. Leave conversation
5. Close your browser windows
6. Open your browser & click on the livechat element again
7. see that no name and email is no more needed until you delete your browser cookies

### Expected behavior:

When a user has used the livechat (with entering his name and email before), closes his brower window and would like to use the livechat again, he has to enter the information again.

### Actual behavior:

When a user close his browser window after using the livechat, the information (name and email) are still available, and doesn't have to enter his name and email again.
Only after a cookie deletion the chat window ask the user for a name and email.

### Server Setup Information:

- Version of Rocket.Chat Server: 0.74.2
- Operating System: CentOS 7.6
- Number of Running Instances: 4
- NodeJS Version: 8.11.4
- MongoDB Version: 3.6.9

**Comment by renatobecker** [comment]
Hi @robinreinhardt, in the issue's description you've mentioned a "problem" related to the behavior you described, but do you really think that it's a "problem" or it's a natural behavior of the browser?
If you really need to change this behavior, I would suggest you to close this issue and create another issue in this repository:

https://github.com/RocketChat/feature-requests

Thanks.
